apps/web/src/app/admin-console/organizations/shared/components/integrations/models.ts

```diff
@@ -21,4 +23,49 @@ export type Integration = {
   isConnected?: boolean;
   canSetupConnection?: boolean;
   configuration?: string;
+  template?: string;
+
+  HecConfiguration?: HecConfiguration | null;
+  HecConfigurationTemplate?: HecConfigurationTemplate | null;
 };
+
+/*
+ * Represents the configuration for a HEC (HTTP Event Collector) integration.
+ * Configuration model that is required by OrganizationIntegration.
+ */
+export class HecConfiguration {
+  uri: string;
+  scheme = "Bearer";
+  token: string;
+  service: OrganizationIntegrationServiceType;
+
+  constructor(uri: string, token: string, service: string) {
+    this.uri = uri;
+    this.token = token;
+    this.service = service as OrganizationIntegrationServiceType;
+  }
+
+  toString(): string {
+    return JSON.stringify(this);
+  }
+}
+
+/**
+ * Represents the configuration template for a HEC (HTTP Event Collector) integration.
+ * from OrganizationIntegrationConfiguration
+ */
+export class HecConfigurationTemplate {
+  event = "#EventMessage";
```

Comment:
```suggestion
  event = "#EventMessage#";
```

Should have `#` on both sides of the variable (otherwise the server won't see this variable token to replace)